3g-watchdog

```diff
@@ -1,3 +1,21 @@
+/*
+3g-watchdog - monitors 3G connectivity and takes action if it's down
+Copyright (C) 2018, The Cacophony Project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
```

Comment:
The header looks great but this isn't a valid comment for a Python program! You'll need to change it to a `#` or triple quote style comment (https://twitter.com/gvanrossum/status/112670605505077248)